Doesn't mark cond as fully covered

JacekLach opened the pull request:
Given a cond where every test and every expr (including `:else` and its expr) is marked as covered, the "head" line is marked as only partially covered.

frankshearar commented:
Same thing happens for `and`:

```
(and ; marked as not covered
  (true thing)
  (another true thing))
```

Commit:
Special case cond to avoid false partial marks

(cond :else 1) expands to (if :else 1 (cond)), with the second branch never
covered. This is misleading (shows orange (cond :else 1) instead of green),
so special case cond.
